product-web-app/package.json

```diff
@@ -8,7 +8,7 @@
     "prestart": "npm run compile && cp -R views dist",
     "compile": "tsc",
     "lint": "npx eslint . --fix  --ext .ts && prettier --write \"**/*.+(ts)\"",
-    "start": "node dist/app.js",
+    "start": "npx tsc && cp -R views dist && node dist/app.js",
```

Comment:
It already happens in "prestart"
https://docs.npmjs.com/cli/v9/using-npm/scripts#pre--post-scripts